### PR TITLE
Adds sample for gfycat

### PR DESCRIPTION
**Pull request by kul3r4**
@sebastianbenz PTAL

### CODE REVIEW

File: src/20_Components/amp-gfycat.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!-- 
+  #### Introduction
+
+  Embed gfycat in your AMP HTML files.
```
Comment (sebastianbenz) [inline src/20_Components/amp-gfycat.html]: can you add a link to gfycat.com

File: src/20_Components/amp-gfycat.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- #### Setup -->
+  <!--
+    Import the `amp-gfycat` library.
+  -->
+  <script async custom-element="amp-gfycat" src="https://cdn.ampproject.org/v0/amp-gfycat-0.1.js"></script>
```
Comment (sebastianbenz) [inline src/20_Components/amp-gfycat.html]: s/library/component/

File: src/20_Components/amp-gfycat.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+    Embed gfycat via the `data-gfyid` attribute. We recommend using responsive layout for better experience. Find a list of all supported layouts [here](https://www.ampproject.org/docs/guides/responsive/control_layout.html).
```
Comment (sebastianbenz) [inline src/20_Components/amp-gfycat.html]: Can you find a better phrase for " Embed gfycat " ? What is it that you're embedding?

File: src/20_Components/amp-gfycat.html (hunk 1 above)
Comment (sebastianbenz) [inline src/20_Components/amp-gfycat.html]: Also add a link to the AMP docs. Maybe rephrase to:

Use the amp-gfycat component to embed animated GIFs from gfycat.

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
Excellent choice for the sample Gif btw.
